webextensions/manifest/background.json

```diff
@@ -146,6 +146,27 @@
               "safari_ios": "mirror"
             }
           }
+        },
+        "type": {
+          "__compat": {
+            "support": {
+              "chrome": {
+                "version_added": true,
+                "notes": "When set to <code>module </code>, includes service workers as ES modules."
```

Comment:
This and Firefox: the note can be removed. The meaning of type:"module" belongs to mdn/content.